### DISCUSSION TITLE
[List User Companies] Inconsistency between documentation and actual response

### DISCUSSION THREAD

**Discussion by Zanfo96** [fattureincloud/api#241] [Q&A] [answered]
Good morning Mauro and everyone,

currently working on APIv2 migration. I have a doubt about the response of the endpoint "List User Companies".

I have a test account on FattureInCloud with some companies; when calling the endpoint, I receive a response that contains way more fields that what is reported in the endpoint documentation or example response.

![image](https://user-images.githubusercontent.com/82850061/214281474-ec6cca5e-c9f9-4bfd-984a-7e32326e8fa2.png)

![image](https://user-images.githubusercontent.com/82850061/214281546-eadee988-02e0-4cdd-a307-bd930151c8de.png)

while the response for my account is something like this

![image](https://user-images.githubusercontent.com/82850061/214281890-ff6459ac-8ad4-4330-9365-701d372667d8.png)

My question about it are:

why do we have this mismatch between actual response and documentation? 

and the main question is: can we trust that "VAT Number" field will be provided regularly in this endpoint or could it be removed in the future? Since we're filtering companies per VAT Number, we depend on this field being provided in the List User Companies endpoint. 

Thank you very much for your time and have a nice day!

**Comment by emajo** [comment, marked as answer]
Hi @Zanfo96, the fields non documented should not be used as they will probably be removed in the future, regarding the `vat_number` field you don't have to worry as it will always be included in the response.

Thanks,
Emanuele.
